### ISSUE TITLE
Inbound call only works once.

### ISSUE THREAD

**Issue by papajowsef** [fusionpbx/fusionpbx#1054]
What steps will reproduce the problem?
1. Install Fusionpbx using the script below.

http://fusionpbx.googlecode.com/svn/branches/dev/scripts/install/ubuntu/install_fusionpbx.sh

Continue with the current script [y/N]? (Choose 'N' this will get the most recent version of the install script)

I typed Y

2. Configured Extension number and works fine

3. Configured SIP account and Dial Plan Destination.

What is the expected output? What do you see instead?

I'm able to call the SIP number and routed to my extension #100 but only once. If I tried another call nothing happen. No logs in the sophia fscli.      

What version of the product are you using? On what operating system?

freeswitch@internal> version
FreeSWITCH Version 1.4.20+git~20150715T215728Z~71ee949431~64bit (git 71ee949 2015-07-15 21:57:28Z 64bit)

OS:  Ubuntu 14.04  AWS EC2 instance

Please provide any additional information below.

I noticed that I need to setup to  global domain first.  After I changed back to the IP address domain, inbound call works fine. But if I tried to call again, nothing happens. No logs in the fscli.  Appreciate your help here.

**Comment by richbreton** [comment]
Google code has been deprecated. Please read the installation instructions here: https://github.com/fusionpbx/fusionpbx

**Comment by papajowsef** [comment]
Thanks Richbreton!

**Comment by richbreton** [comment]
Looks like this was fixed from another ticket #1056 by adding some rules and using the new installer closing the ticket
